pkg/config/config.go

```diff
@@ -281,3 +282,16 @@ func GetDeployOrigin() (src string) {
 func RunningInInstaller() bool {
 	return os.Getenv(model.OktetoInInstaller) == "true"
 }
+
+func IsInsecureSkipVerify() bool {
+	envVarValue := os.Getenv("OKTETO_INSECURE_SKIP_TLS_VERIFY")
```

Comment:
Should we use a constant for the variable? In case we decide to change we have to change it only in one place

Reply:
i think we have it only here, but i agree maybe adding this as we have the other envs at the model pkg, but make it constant of the config pkg ?